overlore/llm/constants.py

```diff
@@ -82,15 +83,39 @@
 )
 
 AGENT_CREATION_TEMPLATE = """
```

Comment:
Can we use the same introduction sentence as for the prompt template: "You are a gamemaster for a game..." ?